### ISSUE TITLE
Internal error when adding training

### ISSUE THREAD

**Issue by jshprentz** [sleede/fab-manager#211]
**Summary:** Adding a new training displays "Internal Server Error" in the administrator's web interface.

**Environment:** Fab Manager 4.4.1 runs in Docker containers on a 4GB AWS Lightsail instance with Ubuntu 18.04.4.

**What the Administrator sees:** After filling out the "Add a New Training" form and clicking "Validate Your Training," these errors display above the form. The letters in the error messages spell "Internal Server Error."

![image](https://user-images.githubusercontent.com/5303785/82001403-2661f180-9629-11ea-965e-a82184a1c212.png)

**Log file:** The app-stdout.log contains these messages about the problem.

```
Started POST "/api/trainings/" for 72.83.101.211 at 2020-05-15 01:21:34 +0000
Processing by API::TrainingsController#create as JSON
  Parameters: {"training"=>{"name"=>"Sample Training", "training_image_attributes"=>{"attachment"=>#<ActionDispatch::Http::UploadedFile:0x00005639302541e8 @tempfile=#<Tempfile:/tmp/RackMultipart20200515-7-1y3j19k.jpg>, @original_filename="Sewing Machines.jpg", @content_type="image/jpeg", @headers="Content-Disposition: form-data; name=\"training[training_image_attributes][attachment]\"; filename=\"Sewing Machines.jpg\"\r\nContent-Type: image/jpeg\r\n">}, "description"=>"<p>Blah blah<br></p>", "machine_ids"=>["4", ""], "nb_total_places"=>"2", "public_page"=>"", "disabled"=>""}, "authenticity_token"=>"MszpB/D2tjjICpXUywpjazemiOte7rdt7gU5hoRc4/TDW9Y2+T+iE51x3zwYSmIwAPZwLJT8esHTe8GwEXFbVw=="}
  ^[[1m^[[36mUser Load (0.3ms)^[[0m  ^[[1m^[[34mSELECT  "users".* FROM "users" WHERE "users"."id" = $1 ORDER BY "users"."id" ASC LIMIT $2^[[0m  [["id", 1], ["LIMIT", 1]]
Completed 500 Internal Server Error in 3ms (ActiveRecord: 0.3ms | Elasticsearch: 0.0ms)

NoMethodError (undefined method `create?' for #<TrainingPolicy:0x00005639302284d0>):

app/controllers/api/trainings_controller.rb:26:in `create'
```

**Context:** I am evaluating Fab Manager for Nova Labs, a makerspace in Reston, Virginia, USA. The demo server has been running for about a day at https://nl-fm-demo.bamboolean.com. The server is configured with the en-us locale and US date formats. I have successfully added the following to the demo Fab Manager:
- Site customization
- About page
- Users, groups, tags
- Spaces
- Events, categories, themes, price categories
- Machines
- Projects, themes

**Comment by sylvainbx** [comment]
Hi @jshprentz,
I confirm the issue and that your PR fixes the problem. The fix will be available with the next release.

**Comment by sylvainbx** [comment]
Fixed with [v4.4.2](https://github.com/sleede/fab-manager/releases/tag/v4.4.2)
